tests/test_examples.py

```diff
@@ -63,10 +83,14 @@ def powerset(iterable):
         ("examples/tutorials/async_rendering.py",),
     ],
 )
-def test_example_modules(args):
+def test_example_modules_no_bullet(args):
```

Comment:
Naming nit. Some of these (e.g. `managed_rigid_object_tutorial`) are optional physics rather than no-physics.
```suggestion
def test_example_modules_optional_bullet(args):
```